tests/phpunit/bootstrap.php

```diff
@@ -0,0 +1,2 @@
+<?php
```

Comment:
I only had it there, because PHPUnit complained when the file was absent. I can put in a `require`-call to `vendor/autoload.php` if you want..